src/pageObject/BasePage.ts

```diff
@@ -1,7 +1,18 @@
 import { Page } from '@playwright/test';
+import { getText } from '../config/locales.js';
 
 export class BasePage {
-    constructor(protected page: Page, private readonly path: string) {}
+    private okCookieButtonLocator;
+
+    constructor(protected page: Page, private readonly path: string) {
+        this.okCookieButtonLocator = this.page.getByText(getText('okCookie'));
+
+        this.page.addLocatorHandler(this.okCookieButtonLocator, async (locator) => {
+            if (await locator.isVisible()) {
```

Comment:
is visible not needed here, if handler captured it means that locator is present on the page